gomplate.go

```diff
@@ -108,15 +117,60 @@ func (g *gomplate) runTemplate(t *tplate) error {
 	return err
 }
 
+type templateAliases map[string]string
+
 // newGomplate -
-func newGomplate(d *data.Data, leftDelim, rightDelim string) *gomplate {
+func newGomplate(d *data.Data, leftDelim, rightDelim string, ta templateAliases) *gomplate {
 	return &gomplate{
-		leftDelim:  leftDelim,
-		rightDelim: rightDelim,
-		funcMap:    Funcs(d),
+		leftDelim:       leftDelim,
+		rightDelim:      rightDelim,
+		funcMap:         Funcs(d),
+		templateAliases: ta,
 	}
 }
 
+func parseTemplateArgs(templateArgs []string) (templateAliases, error) {
```

Comment:
the linter's annoyed about this function having high cyclomatic complexity:

```
gomplate.go:132::warning: cyclomatic complexity 11 of function parseTemplateArgs() is high (> 10) (gocyclo)
```

If you see a simple way to refactor this to lower the complexity, that'd be great, otherwise, just add `// nolint: gocyclo` above this line and I'll maybe get to simplifying this _later_... 😉